geolocation-API/disabled-by-permissions-policy.https.sub.html

```diff
@@ -10,9 +10,9 @@
     "use strict";
 
     const same_origin_src =
-      "/feature-policy/resources/feature-policy-geolocation.html";
+      "/feature-policy/resources/permissions-policy-geolocation.html";
     const cross_origin_src =
-      "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
```

Comment:
At least on the version of wptserve Chromium is using this causes the server to throw an exception. Are you sure this is the correct syntax? What was wrong with the existing code?

Reply:
oops, indeed... should have been https://{{hosts[][]}}